jlm/tooling/CommandLine.cpp

```diff
@@ -253,11 +253,9 @@ const std::unordered_map<JlmOptCommandLineOptions::OutputFormat, std::string_vie
 JlmOptCommandLineOptions::GetOutputFormatCommandLineArguments()
 {
   static std::unordered_map<OutputFormat, std::string_view> mapping = {
-    { OutputFormat::Ascii, "ascii" },
-    { OutputFormat::Llvm, "llvm" },
-    { OutputFormat::Mlir, "mlir" },
-    { OutputFormat::Tree, "tree" },
-    { OutputFormat::Xml, "xml" }
+    { OutputFormat::Ascii, "ascii" }, { OutputFormat::Dot, "dot" },
```

Comment:
This appears to be what `clang-format` prefers, at least with our current config.